database/queries/get_liked_careers.js

```diff
@@ -0,0 +1,13 @@
+const dbConnection = require('../db_connection');
+
+const getLikedCareers = callback => {
```

Comment:
This query is going to have to take a `userId` argument at some point so it may as well now. That way when you add the user next week you can just replace the hard-coded argument with the actual `userId`